build-index.sh

```diff
@@ -168,6 +174,12 @@ function list_dir() {
         fi
 	      diff=$(rfcdiff "https://tools.ietf.org/id/${file}.txt" "$this_githubio")
         td "$(a "$diff" 'diff with last submission' diff "$file")"
+        this_issue_label=$(./lib/extract-frontmatter.py ${file}.md github-issue-label)
+        if [[ "$this_issue_label" ]]; then
```

Comment:
You don't need to go looking for a label if `[[ "${#files[@]}" -eq 1 ]]`.  The issues link at the top of the page is enough in that case.